docs/oracles/iot-proof-of-coverage-oracles.mdx

```diff
@@ -0,0 +1,101 @@
+---
+id: iot-proof-of-coverage-oracles
+title: IOT Proof of Coverage Oracles
+pagination_label: IOT Proof of Coverage Oracles
+sidebar_label: IOT Proof of Coverage Oracles
+description: Helium IOT Proof of Coverage Oracles Documentation
+image: https://i.imgur.com/bWAbESE.png
+slug: /oracles/iot-proof-of-coverage-oracles
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+:::warning 🚧 This document is a work in progress. 🚧
+
+The documentation on Solana and the upcoming Migration is under development and subject to change.
```

Comment:
some of this stuff is active now. might be worth highlighting the parts that are.

Reply:
on it! breaking into smaller PRs will make this easier